### PR TITLE
TACKLE-273: Business service and description field population now optional

### PR DESCRIPTION
**Pull request by m-brophy**
https://issues.redhat.com/browse/TACKLE-273

### CODE REVIEW

File: src/test/java/io/tackle/applicationimporter/ImportServiceTest.java [hunk 1]
```diff
@@ -368,86 +368,7 @@ protected void testMultipartImport() throws SystemException, NotSupportedExcepti
...
-        businessService.id = "1";
-        businessService.name = "BS 2";
-        businessServices.add(businessService);
-        svc.mapImportsToApplication(appList, tags, businessServices, appImportParent);
-
-
-        userTransaction.commit();
-
-        ApplicationImport refusedImport1 = ApplicationImport.findById(id);
-        assertEquals(Boolean.FALSE, refusedImport1.getValid());
```
Comment (mrizzi) [inline src/test/java/io/tackle/applicationimporter/ImportServiceTest.java:430]: Could we keep this test and just change the expected outcome to check missing mandatory fields are allowed?

**Review by mrizzi — COMMENTED** [review]
Suggestion for "recycling" the test rather than remove it.

File: src/test/java/io/tackle/applicationimporter/ImportServiceTest.java (hunk 1 above)
Comment (m-brophy) [reply src/test/java/io/tackle/applicationimporter/ImportServiceTest.java:430]: I've done this now

**Review by mrizzi — APPROVED** [review]
manual tests have been fine and suggested change has been applied.
Thanks @m-brophy :+1:

**Pull request merged**